src/python/foglamp/__main__.py

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+
+# FOGLAMP_BEGIN
+# See: http://foglamp.readthedocs.io/
+# FOGLAMP_END
+
+import foglamp.core.core_daemon
```

Comment:
I tried core_daemon being renamed to daemon. It hides the third-party daemon package.